frontend/src/templates/Field/Nric/NricField.tsx

```diff
@@ -1,7 +1,7 @@
 /**
  * @precondition Must have a parent `react-hook-form#FormProvider` component.
  */
-import { useMemo, useState } from 'react'
+import { useMemo } from 'react'
```

Comment:
what's this change? haha if its removing an unused variable, maybe we can include this in a different PR into develop.

Reply:
We could establish a new norm for those 🤔. We see "somewhat often" the unused variables issues reported as warnings by github in the PRs. That's why we've been clearing them in the same stride, but you are right: that pollutes the file list on a given PR. We should instead spend 5 more mins to open trivial lint PRs just to clear those. (Well.. 5 mins to make the lint PRs, 30mins to wait for tests and merge 😢)

Reply:
Yea it's just to remove the unused import. I'll undo this change on this PR and start a new PR if it's still there on the `feat/payment-mvp` branch then.

Reply:
Actually is more that I feel these should be done cleanly on the most upstream branch where the change occurred. Since NRIC field wasn't touched by payments I'm p sure it's also there on develop. We should open a PR to develop to fix it haha

Reply:
reverted in 23e38d90

Reply:
Okay, just checked the code on `develop`, this has been removed already!